src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -172,7 +186,14 @@
       (pretty-print-asset symbol value token)]
      " "
      [react/text {:accessibility-label :currency-text}
-      (clojure.string/upper-case (name symbol))]]
+      (clojure.string/upper-case (name symbol))]
+     " ("
+     [react/text {:accessibility-label :amount-text}
+      (pretty-print-fiat transaction currency prices)]
```

Comment:
the logic should be moved in the `:wallet.transactions/transaction-details` subscription